test/translation/lua/dotColonFunctionCalls.lua

```diff
@@ -1,6 +1,6 @@
 classInstance:colonMethod();
-classInstance.dotMethod();
+classInstance:dotMethod();
```

Comment:
We should probably get rid of this translation test. It's fully covered by functional tests and doesn't even make sense with how functions are transpiled now.